app/controllers/teams_controller.rb

```diff
@@ -6,11 +6,19 @@ class TeamsController < ApplicationController
   load_and_authorize_resource except: [:index, :show]
 
   def index
```

Comment:
I accept suggestions for refactoring this function.

My brain is blocked for now. :crying_cat_face: